scipy/signal/ltisys.py

```diff
@@ -578,8 +578,8 @@ class TransferFunction(lti):
     """Linear Time Invariant system class in transfer function form.
 
     Represents the system as the transfer function
-    :math:`H(s)=\sum_i b[i] s^i / \sum_j a[j] s^i`, where :math:`a` are
-    elements of the numerator `num` and :math:`b` are the elements of the
+    :math:`H(s)=\sum_{i=0}^N b[N-i] s^i / \sum_{j=0}^M a[M-j] s^i`, where :math:`b` are
```

Comment:
The term in the summation in the denominator should be `s^j`, or the dummy variable in the should be changed to `i`.